identifiers/country-nl/electoral_districts.csv

```diff
@@ -0,0 +1,21 @@
+id,name
+ocd-division/country:nl/ed:01,Groningen
```

Comment:
Are the district numbers tied to the identity of the district or are they arbitrarily assigned? In the past we've seen issues with the OCD IDs for a country when district numbers change after redistricting.

If it's not tied tightly with the identity, I'd recommend using the name of the district instead of the district number.

Reply:
"Nummer van de kieskring" = "Constituency number", so it at least isn't just a numbered list (as in, here are 20 constituencies). I don't know if they are consistent over time, however.

Reply:
Ack, I'll wait for a response from @sguenther85 on this, but assuming these appear to be consistent over time I'm fine leaving them as is. I'd just like to avoid having OCD IDs  with `ed:xx-yyyy` where possible

Reply:
The number is specifically assigned to each constituency. Source:
https://www.kiesraad.nl/verkiezingen/adviezen-en-publicaties/formulieren/2016/5/kieskringen/kieskringen-en-hoofdstembureaus-tweede-kamerverkiezingen-en-europees-parlementsverkiezingen

Reply:
Also, we have seen the same numbering in the candidate list for the election

Reply:
I can't find any records of redistricting so sticking with the numbered districts as assigned sounds good to me.